test/server/rooms/active-room.spec.js

```diff
@@ -265,4 +267,175 @@ describe('active-room', function() {
         });
 
     });
+    
+    let defaultConfig = {
+        name: 'test',
+        owner: 'alice',
+        roles: {
+            role1: ['alice'],
+            role2: ['bob', 'eve']
+        }
+    };
+    
+    describe('without projects', function() {
+        let room = null;
+        let alice, bob;
+        before(function() {
+            room = utils.createRoom(defaultConfig);
+            alice = room.getSocketsAt('role1')[0];
+            bob = room.getSocketsAt('role2')[0];
+        });
+        
+        describe('remove', function() {
+            it('should remove a socket', function() {
+                room.remove(alice);
+                assert.equal(room.roles.role1.length, 0);
```

Comment:
I would use `getSocketAt` instead of this so changing the internals doesn't break the tests.